src/sentry/api/endpoints/organization_events_trace.py

```diff
@@ -17,20 +20,83 @@
 MAX_TRACE_SIZE = 100
 NODESTORE_KEYS = ["timestamp", "start_timestamp"]
 
+SnubaEvent = Mapping[str, str]
+TraceError = Mapping[str, str]
+ResponseType = Dict[str, Any]
+
+
+class TraceEvent:
```

Comment:
this class doesn't do a whole lot of stuff either, may also consider typeddict here but it's not like this is any worse. just keeps the diff smaller